front-end/studio/src/app/pages/apis/{apiId}/editor/editor.component.ts

```diff
@@ -193,14 +193,24 @@ export class ApiEditorComponent implements OnChanges, OnInit, OnDestroy, IEditor
      */
     public onGlobalKeyDown(event: KeyboardEvent): void {
         // TODO skip any event that was sent to an input field (e.g. input, textarea, etc)
-        if (event.ctrlKey && event.key === 'z' && !event.metaKey && !event.altKey) {
+        if (ApiEditorComponent.isUndo(event)) {
             this.undoLastCommand();
         }
-        if (event.ctrlKey && event.key === 'y' && !event.metaKey && !event.altKey) {
+        if (ApiEditorComponent.isRedo(event)) {
             this.redoLastCommand();
         }
     }
 
+    private static isUndo(event: KeyboardEvent): boolean {
```

Comment:
Would you prefer these in a utility/helper class somewhere? If so, where?

Reply:
Also, leaving them here seems completely reasonable to me, but just wanted to check :)